build/karma.conf.js

```diff
@@ -55,7 +55,7 @@ module.exports = function(config) {
 
     logLevel: 'WARN',
 
-    frameworks: ['jasmine', 'browserify'],
+    frameworks: ['jasmine-jquery', 'jasmine', 'browserify'],
```

Comment:
Do you know why is that? Is it now possible to use jQuery in tests? 

(I'm asking this because I'm afraid that ppl start using jQuery in tests when it is available)